sjb/config/test_cases/test_branch_origin_extended_conformance_install_update.yml

```diff
@@ -23,6 +23,10 @@ extensions:
       script: |-
         tito_tmp_dir="tito"
         mkdir -p "${tito_tmp_dir}"
+        if [[ ${ORIGIN_TARGET_BRANCH} == release-1.* ]]
+        then
+          git checkout -b ${ORIGIN_TARGET_BRANCH} origin/${ORIGIN_TARGET_BRANCH}
```

Comment:
Isn't there a bunch of logic for translating branch names? Where Origin has `release-1.x` or `release-3.x` and OpenShift Ansible has `enterprise-x`?